crates/turbopack-ecmascript/src/references/esm/binding.rs

```diff
@@ -19,6 +19,20 @@ use crate::{
     references::AstPath,
 };
 
+#[turbo_tasks::value(shared)]
+#[derive(Hash, Debug)]
+pub struct EsmBindings {
+    pub bindings: Vc<Vec<Vc<EsmBinding>>>,
+}
+
+#[turbo_tasks::value_impl]
+impl EsmBindings {
+    #[turbo_tasks::function]
+    pub fn new(bindings: Vc<Vec<Vc<EsmBinding>>>) -> Vc<Self> {
```

Comment:
The main idea of this optimization was to avoid calling a turbo tasks function per EsmBinding. So it would be better to store a Vec<EsmBinding> instead of Vec<Vc<EsmBinding>>

Reply:
How can I store vector here?

```
`turbo_tasks::Vc<std::vec::Vec<references::esm::binding::EsmBinding>>` is not a future
the trait `std::future::IntoFuture` is not implemented for `turbo_tasks::Vc<std::vec::Vec<references::esm::binding::EsmBinding>>`
turbo_tasks::Vc<std::vec::Vec<references::esm::binding::EsmBinding>> must be a future or must implement `IntoFuture` to be awaited

```

Reply:
Done